physionet-django/notification/templates/notification/email/notify_credential_reference_response_reminder.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}{% autoescape off %}{% filter wordwrap:70 %}
+Dear {{ applicant_name }},
+
+This is to remind you that your application for credentialed access to {{ SITE_NAME }} is still pending. We are currently waiting from your reference {{ application.reference_name }} to submit their response.
+
+We contacted your reference on {{ application.reference_contact_datetime }}
+
+Please follow up with your reference so that they can submit their response as soon as possible.
+
+Please note that your application will be automatically rejected if we do not receive a response from your reference within {{ auto_rejection_days }} days of the date of this email.
```

Comment:
"Rejected" doesn't sound very friendly, so I prefer to avoid saying this where we can. In this case, perhaps "closed" would be a good alternative? "`...will be closed if...`".

Reply:
Ah yes, my bad, i remember you had pointed this out that the wording is negative, on of the previous PR.